aspnetcore/signalr/get-started-signalr-core.md

```diff
@@ -46,30 +58,60 @@ Install the following software:
 
   ![New Project dialog in Visual Studio](get-started-signalr-core/_static/signalr-new-project-choose-type.png)
 
-  The libraries that host SignalR server-side code are included in the project template. Install the client-side JavaScript separately with [npm](https://www.npmjs.com/).
+The libraries that host SignalR server-side code are included in the project template. Install the client-side JavaScript separately with [npm](https://www.npmjs.com/).
 
   ```console
    npm install @aspnet/signalr
   ```
 
 3. Copy the *signalr.js* from *node_modules\\@aspnet\signalr\dist\browser* to the *wwwroot\lib* folder in your project.
 
+# [Visual Studio Code](#tab/visual-studio-code)
+
+1. From a terminal, run the following commands:
+ 
+  ```console
+  mkdir SignalRChat
+  cd SignalRChat
+  dotnet new SignalRChat razor
+  ```
+
+-----
+
 ## Create the SignalR Hub
 
 A hub is a class that serves as a high-level pipeline that allows the client and server to call methods on each other.
 
-1. Add a class to the project by choosing **File** > **New** > **File** and selecting **Visual C# Class**. 
+# [Visual Studio](#tab/visual-studio)
+
+1. Add a class to the project by choosing **File** > **New** > **File** and selecting **Visual C# Class**.
 
 1. Inherit from `Microsoft.AspNetCore.SignalR.Hub`. The `Hub` class contains properties and events for managing connections and groups, as well as sending and receiving data.
 
-1. Create the `Send` method that sends a message to all connected chat clients. Notice it returns a `Task`, because SignalR is asynchronous. Asynchronous code scales better.
+1. Create the `SendMessage` method that sends a message to all connected chat clients. Notice it returns a `Task`, because SignalR is asynchronous. Asynchronous code scales better.
```

Comment:
Link to the Task API instead of code-fencing it:
`[Task](/dotnet/api/system.threading.tasks.task)`